specs/card.proto

```diff
@@ -39,10 +35,20 @@ message StartDayExtension {
   optional uint32 start_day = 1;
 }
 
+message NuernbergPassIdExtension {
+  optional uint32 pass_id = 1;
+}
+
+enum NuernergPassIdentifier {
+  passNr = 0;
+  passId = 1;
+}
+
 message CardExtensions {
   optional RegionExtension extension_region = 1;
   optional BirthdayExtension extension_birthday = 2;
-  optional NuernbergPassNumberExtension extension_nuernberg_pass_number = 3;
+  optional NuernbergPassIdExtension extension_nuernberg_pass_id = 3;
+  optional NuernergPassIdentifier nuernberg_pass_identifier = 30;
```

Comment:
should this be a separate extension? I expected this to  be added to the `NuernbergPassIdExtension`